AUTHORS

```diff
@@ -1,3 +1,6 @@
 Timur Shemsedinov <timur.shemsedinov@gmail.com>
+Dimon Durak <Dimon Durak>
```

Comment:
@dimon-durak can you please provide your real name and email if you want to be included to the list?